src/stories/Inputs.stories.ts

```diff
@@ -0,0 +1,53 @@
+import { fn } from '@storybook/test';
+
+import { Input } from '../components/inputs-dropdowns/Input';
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+export default {
+  title: 'Example/Input',
+  component: Input,
+  parameters: {
+    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/configure/story-layout
+    layout: 'centered',
+  },
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/writing-docs/autodocs
+  tags: ['autodocs'],
+  // More on argTypes: https://storybook.js.org/docs/api/argtypes
+  argTypes: {
+    backgroundColor: { control: 'color' },
+    suggestions: {
+      control: { type: 'object' },
+      description: 'Autocomplete suggestions',
+      defaultValue: ['abacus', 'boy', 'car'],
+    }
+  },
+  // Use `fn` to spy on the onClick arg, which will appear in the actions panel once invoked: https://storybook.js.org/docs/essentials/actions#action-args
+  args: { onClick: fn(),
```

Comment:
![image](https://github.com/user-attachments/assets/e690ed59-92fc-40f7-beec-c53c48a2869d)

Maybe change `onClick` to `onChange`, both so that it removes the onClick row and also so it shows up in the actions panel (if that's real)